### PR TITLE
chore(deps): update deps

### PR DESCRIPTION
**Pull request by anshumanv**
update deps

**What kind of change does this PR introduce?**
updated deps

**Did you add tests for your changes?**

**If relevant, did you update the documentation?**
NA

**Summary**
Updated deps

**Other information**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,54 +132,54 @@
...
     "rimraf": "^2.6.2",
     "schema-utils": "^1.0.0",
     "ts-jest": "^23.10.5",
     "ts-node": "^7.0.1",
     "typedoc": "^0.14.2",
-    "typescript": "^3.3.1",
-    "webpack": "4.x.x",
-    "webpack-dev-server": "^3.3.1"
+    "typescript": "^3.5.1",
+    "webpack": "^4.33.0",
```
Comment (anshumanv) [inline package.json]: Do we leave this as `4.x.x`?

**Comment by anshumanv** [comment]
Looking into CI

File: package.json (hunk 1 above)
Comment (evenstensberg) [reply package.json]: yes

**Comment by anshumanv** [comment]
On it @ematipico 
Meanwhile can you help figure out the error, works fine locally. 🤔

**Comment by dhruvdutt** [comment]
Duplicate of #907
Already being worked upon.
There's some issue with the latest version of Node. Trying to figure that out.

**Comment by anshumanv** [comment]
I don't see any real upgrades, you just pinned the deps 🤔

**Comment by dhruvdutt** [comment]
https://github.com/webpack/webpack-cli/pull/907#issuecomment-498296812

**Pull request closed without merging**